### PR TITLE
WIP: Copy edits to the FAQ

### PR DESCRIPTION
**Pull request by meyerweb**
Please review the changes for accuracy.  Thank you!

----

Site preview: https://igalia.github.io/wpewebkit.org/issue-69/

### CODE REVIEW

File: about/faq.md [hunk 1]
```diff
@@ -23,134 +19,93 @@ platform, as well as different fixes for its supported devices. Quoting the
...
-platforms and products.
+Cog's usage scenarios span from a MiniBrowser application to a full web-app container application meant to run HTML-based user interfaces on embedded platforms and products.
 
-Although it can run on Linux-based desktop environments, Cog is not a full-blown
-Web Browser to be compared with Google Chrome or Safari. Cog's primary
-environment is on embedded platforms, it can run within a Wayland compositor
-such as Weston. Additionally if the platform supports KMS/DRM, Cog can run as a
-full-screen standalone browser, this use-case is very common on kiosk products
-for instance.
+Although it can run on Linux-based desktop environments, Cog is not a full-blown Web Browser to be compared with Google Chrome or Safari. Cog's primary environment is on embedded platforms, and it can run within a Wayland compositor such as Weston. Additionally, if the platform supports KMS/DRM, Cog can run as a full-screen standalone browser, this use-case is very common on kiosk products for instance.
```
Comment (bkardell) [inline about/faq.md:83]: Out of curiosity, what does KMS/DRM stand for in this? I don't know what KMS is at all and I would've intuited DRM is digital rights management, but that doesn't make a lot of sense here in this context to me at least.

Comment (meyerweb) [reply about/faq.md:83]: I actually don’t know myself, but I wasn’t sure how much acronym expansion I should be doing in contexts where it was highly likely anyone interested in answer would already know.

Comment (meyerweb) [reply about/faq.md:83]: Turns out in this case:

> KMS stands for Kernel Mode Setting and is a sub-part of the DRM API
> DRM stands for Direct Rendering Manager and was introduced to deal with graphic cards embedding GPUs

(From https://events.static.linuxfound.org/sites/events/files/slides/brezillon-drm-kms.pdf)

File: about/faq.md [hunk 2]
```diff
@@ -23,134 +19,93 @@ platform, as well as different fixes for its supported devices. Quoting the
...
-The WPE project currently does not have an official list of the Web features it
-supports. We might provide one, but for the time being we recommend users to
-check for specific features themselves by testing WPE through the Cog browser.
 
 ## What's the status regarding WebRTC? 
 
 As of end of 2020 the facts are:
 
-1. [WPEWebKit upstream has support for WebRTC](https://blogs.gnome.org/tsaunier/2018/07/31/webkitgtk-and-wpe-gains-webrtc-support-back/), by relying on LibWebRTC.
+1. [WPEWebKit upstream has support for WebRTC](https://blogs.gnome.org/tsaunier/2018/07/31/webkitgtk-and-wpe-gains-webrtc-support-back/), by relying on [LibWebRTC](https://github.com/aisouard/libwebrtc).
```
Comment (philn) [inline about/faq.md:46]: The upstream libwebrtc code is hosted on https://webrtc.googlesource.com/src/

**Review by philn — CHANGES_REQUESTED** [review]
Just a small change requested. Also, really a minor thing, I was wondering why this PR rewraps most paragraphs to be wider?

We'd need to adopt a default  config at some point, perhaps using https://editorconfig.org/

File: about/faq.md (hunk 2 above)
Comment (meyerweb) [reply about/faq.md]: Link updated and pushed.

File: about/faq.md [hunk 3]
```diff
@@ -23,134 +19,93 @@ platform, as well as different fixes for its supported devices. Quoting the
 The [RDK WPE page](https://wiki.rdkcentral.com/display/RDK/WPE) provides more
 information about WPE in the RDK platform.
 
-In 2017 engineers from Igalia submitted a new flavor of WPE, suitable for upstream
-hosting under the webkit.org umbrella. This version of WPE its [released every 6
-months](/release/schedule) from the code hosted on the [upstream repository](https://webkit.org).
-This flavor of WPE is maintained upstream and provides regular [security updates](/security).
+In 2017, engineers from [Igalia](https://igalia.com) submitted a new flavor of WPE, suitable for upstream hosting under the [webkit.org](https://webkit.org/) umbrella. This version of WPE is [released every 6 months](/release/schedule) from the code hosted on the [upstream repository](https://webkit.org). This flavor of WPE is maintained upstream and provides regular [security updates](/security).
```
Comment (meyerweb) [inline about/faq.md:22]: A question: is the `[upstream repository]` link best pointed to `https://webkit.org` or is there a better destination to use?

Comment (philn) [reply about/faq.md:22]: And the project plans to migrate to GitHub, that might be a better, future-proof link as well.

Comment (philn) [reply about/faq.md:22]: https://github.com/WebKit/WebKit

Comment (meyerweb) [reply about/faq.md]: Link updated and pushed.

**Comment by philn** [comment]
OK this could go in but can you rebase and squash all commits in one?

**Comment by meyerweb** [comment]
> OK this could go in but can you rebase and squash all commits in one?

I don’t know, I don’t think I’ve ever done that before.  Happy to with pointers to the best way to do that, though!

**Comment by philn** [comment]
Maybe try to follow this doc? https://docs.github.com/en/github/using-git/using-git-rebase-on-the-command-line
Otherwise we could try to setup an interactive remote shell with https://tmate.io/

**Comment by meyerweb** [comment]
> Maybe try to follow this doc? https://docs.github.com/en/github/using-git/using-git-rebase-on-the-command-line

I think I found another way, thanks to https://stackoverflow.com/a/5201642:

```
git reset --soft HEAD~5 && 
git commit --edit -m"$(git log --format=%B --reverse HEAD..HEAD@{1})"
```

That would squash the last five commits (the ones I’ve made so far to this PR) into a single commit, apparently without doing violence to the history and thus (I think) not requiring a force push.  Sound like the right thing to do?

**Comment by meyerweb** [comment]
Done and done, I believe!

**Review by philn — APPROVED** [review]

**Comment by philn** [comment]
Thanks!

**Pull request merged**